lib/components/src/theme.js

```diff
@@ -36,8 +42,16 @@ export const normal = {
   overlayBackground:
     'linear-gradient(to bottom right, rgba(233, 233, 233, 0.6), rgba(255, 255, 255, 0.8))',
 
-  brand: {},
+  brand: {
```

Comment:
So brand is the way you theme/ overload the logo, if you set this to `null` the text will be displayed